scheduler/pkg/store/utils/utils.go

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
note these helpers are tests in experiment and pipeline `db_test.go` via their wrappers.